OpenAI-DotNet/Chat/ChatRequest.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection.Metadata.Ecma335;
```

Comment:
What is this thing used for?

Reply:
no idea, happened automatically somehow.